src/main/java/book/store/repository/book/specification/PriceSpecificationProvider.java

```diff
@@ -0,0 +1,23 @@
+package book.store.repository.book.specification;
+
+import book.store.model.Book;
+import book.store.repository.SpecificationProvider;
+import java.util.Arrays;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PriceSpecificationProvider implements SpecificationProvider<Book> {
+    private static final String FIELD_NAME = "price";
+
+    @Override
+    public Specification<Book> getSpecification(String[] params) {
+        return (root, query, criteriaBuilder) ->
+                root.get(FIELD_NAME).in(Arrays.stream(params).toArray());
```

Comment:
It would be impossible to use this filter for user
We should have min/max (or both) price and search in the range